app/services/ember-cli.js

```diff
@@ -80,7 +80,8 @@ const availableBlueprints = {
 const requiredDependencies = [
   'jquery',
   'ember',
-  'ember-template-compiler'
+  'ember-template-compiler',
+  'ember-data'
```

Comment:
I am slightly concerned with making `ember-data` required.  It is probably fine, but should be considered independently...